### PR TITLE
Added POS apps to onboarding-merchants.md

### PR DESCRIPTION
**Pull request by iBobik**

### CODE REVIEW

File: src/general/onboarding-merchants.md [hunk 1]
```diff
@@ -23,7 +23,8 @@
 There are a number of merchant solutions to help vendors take payment in bitcoin and optionally convert to local fiat:
 
 - [Coinos](https://coinos.io/)
-- [Wallet of Satoshi](https://www.walletofsatoshi.com/)
+- [Wallet of Satoshi POS](https://www.walletofsatoshi.com/pos)
```
Comment (secondl1ght) [inline src/general/onboarding-merchants.md:26]:
```suggestion
- [Wallet of Satoshi](https://www.walletofsatoshi.com/pos)
```
I think most of these are POS's so don't think we need that in the names.

File: src/general/onboarding-merchants.md [hunk 2]
```diff
@@ -23,7 +23,8 @@
...
+- [Breez POS](https://breez.technology/mobile/) (non-custodial)
```
Comment (secondl1ght) [inline src/general/onboarding-merchants.md:27]:
```suggestion
- [Breez](https://breez.technology/mobile/)
```
Let's either remove `(non-custodial)` or add that tag to any of the other options that are also non-custodial.

**Review by secondl1ght — COMMENTED** [review]
Thanks for the PR!

File: src/general/onboarding-merchants.md (hunk 1 above)
Comment (iBobik) [reply src/general/onboarding-merchants.md:26]: In this case "POS" is part of the app name and I think it is good to diferenciace it to the original WoS app.

File: src/general/onboarding-merchants.md (hunk 2 above)
Comment (iBobik) [reply src/general/onboarding-merchants.md:27]: I think it would be good to mention it, but I does not know most of the others.

**Pull request merged**